### ISSUE TITLE
yaxis name alignment fail

### ISSUE THREAD

**Issue by wyrj** [apache/echarts#8444]
### One-line summary [问题简述]
It seems always be 'center' no matter what value I set for yAxis.nameTextStyle.align

### Version & Environment [版本及环境]
+ ECharts version [ECharts 版本]: 4.1.0
+ Browser version [浏览器类型和版本]: 66.0.3359.181
+ OS Version [操作系统类型和版本]: Windows 7

### ECharts option [ECharts配置项]

```javascript
option = {
    xAxis: {
        type: 'category',
        data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
    },
    yAxis: {
        type: 'value',
        name: 'WWW',
        nameTextStyle: {
            color: 'red',
            align: 'left', //the setting is failed
        }
    },
    series: [{
        data: [820, 932, 901, 934, 1290, 1330, 1320],
        type: 'line'
    }]
};

```

**Comment by xellafe** [comment]
Have the same problem
ECharts version [ECharts 版本]: 4.0.4

**Comment by Ovilia** [comment]
Fixed since 4.3.0.
